float32 joint_1
float32 joint_2
float32 joint_3
float32 joint_4
float32 left_gripper_joint
---
geometry_msgs/Pose pose
